src/NuGet/Microsoft.Net.Compilers.Toolset/AnyCpu/build/Microsoft.Net.Compilers.Toolset.props

```diff
@@ -2,17 +2,37 @@
 <Project>
 
   <PropertyGroup>
-    <_RoslynTargetDirectoryName Condition="'$(MSBuildRuntimeType)' == 'Core'">netcore</_RoslynTargetDirectoryName>   
-    <_RoslynTargetDirectoryName Condition="'$(MSBuildRuntimeType)' != 'Core'">net472</_RoslynTargetDirectoryName>   
-    <_RoslynTasksDirectory>$(MSBuildThisFileDirectory)..\tasks\$(_RoslynTargetDirectoryName)\</_RoslynTasksDirectory>
+    <!--
+      Even on netfx hosts we want to use netcore compiler to be consistent with SDK's behavior since .NET 10.
+      The opt out is also the same as in SDK - setting RoslynCompilerType=Framework.
+    -->
+    <_UseRoslynBridgeTask Condition="'$(MSBuildRuntimeType)' != 'Core' and '$(RoslynCompilerType)' != 'Framework'">true</_UseRoslynBridgeTask>
```

Comment:
Is the user meant to be able to put `<RoslynCompilerType>Framework</RoslynCompilerType>` in their project, in addition to PackageReference to MS.Net.Compilers.Toolset, and expect the toolset compiler to run on framework?